spark/src/main/scala/geotrellis/spark/filter/Implicits.scala

```diff
@@ -36,4 +39,10 @@ trait Implicits {
     V,
     M[_]
   ](val self: RDD[(K, V)] with Metadata[M[K]]) extends SpaceTimeToSpatialMethods[K, V, M]
+
+  implicit class withSpaceTimeToSpatialMethods2[
```

Comment:
Well, it looks like it's not the best name for implicit class :D even taking in account  that it will be removed.